### PR TITLE
[LWG 13] P2871R3 Remove deprecated <codecvt> header

### LINKED ISSUE #6671: [2023-11 LWG Motion 13] Remove Deprecated Unicode Conversion Facets from C++26
https://wiki.edg.com/pub/Wg21kona2023/StrawPolls/p2871r3.pdf

### PR DESCRIPTION
**Pull request by AlisdairM**
Fixes #6671
Fixes cplusplus/papers#1527

### CODE REVIEW

**Review by jensmaurer — CHANGES_REQUESTED** [review]

File: source/compatibility.tex [hunk 1]
```diff
@@ -64,6 +64,30 @@
                                 // changed according to the global C locale
 \end{codeblock}
 
+\rSec2[diff.cpp23.depr]{\ref{depr}: compatibility features}
+
+\nodiffref
+\change
+Remove header \tcode{<codecvt>} and all its contents.
```
Comment (jensmaurer) [inline source/compatibility.tex:71]: Do we maybe want to retain an index entry for the mention of the header name?
If someone is looking for that header in a newer standard, the index should point here.
-> use `\libnoheader`

**Review by jensmaurer — CHANGES_REQUESTED** [review]

**Comment by jensmaurer** [comment]
There are overfull `\hbox`es here:

::error file=compatibility.tex::Overfull \hbox (20.8246pt too wide) in paragraph at lines 1933--1968
::error file=compatibility.tex::Overfull \hbox (0.50548pt too wide) in paragraph at lines 1933--1968

Please fix.

File: source/compatibility.tex (hunk 1 above)
Comment (tkoeppe) [reply source/compatibility.tex:171]: Done!

**Comment by tkoeppe** [comment]
> There are overfull `\hbox`es here:
> 
> ::error file=compatibility.tex::Overfull \hbox (20.8246pt too wide) in paragraph at lines 1933--1968 ::error file=compatibility.tex::Overfull \hbox (0.50548pt too wide) in paragraph at lines 1933--1968
> 
> Please fix.

Done.

**Pull request merged**